README.md

```diff
@@ -97,6 +97,34 @@ CONVERSATION_PASSWORD=<add_conversation_password>
 WORKSPACE_ID=<add_conversation_workspace>
 ```
 
+### New API authentication process
+
+The Watson Assistant service has a new API authentication process for service instances that are hosted in the following regions:
+
+* Washington, DC (us-east) as of 14 June 2018
```

Comment:
I would not document history here.  And if we can avoid it, don't mention regions.  Describe what to look for in creds and what to copy.